src/lando/api/views.py

```diff
@@ -1 +1,111 @@
-# Create your views here.
+from functools import wraps
+from typing import Callable
+
+from django import forms
+from django.http import JsonResponse
+from django.views import View
+
+from lando.main.models.revision import DiffWarning, DiffWarningStatus
+from lando.utils.phabricator import get_phabricator_client
+
+
+def phabricator_api_key_required(func: callable) -> Callable:
+    """A simple wrapper that checks for a valid Phabricator API token."""
+
+    @wraps(func)
+    def _wrapper(self, request, *args, **kwargs):
+        HEADER = "X-Phabricator-Api-Key"
```

Comment:
Should it be `HTTP_X_Phabricator_API_Key` now, as in the test fixtures? https://github.com/mozilla-conduit/lando/pull/197/files#diff-3f084e2a5f612fc35bc5c68a611d1ad8959f632315c6e9115149298061dcc371R15

Reply:
These are case-insensitive, but good to change it for consistency .